unit2.js

```diff
@@ -0,0 +1,94 @@
+// function getOption() {
+//     var obj = document.getElementById("mySelect");
+//     document.getElementById("demo").innerHTML = 
+//     obj.options[obj.selectedIndex].text;
+//   }
+
+
```

Comment:
JS file is missing eventListeners such as "DOMContentLoaded" or any other event listeners.